MsixCore/msixmgr/msixmgr.cpp

```diff
@@ -389,9 +625,18 @@ int main(int argc, char * argv[])
 
                 if (FAILED(hrCreateCIM))
                 {
+                    errorCode = msixmgr::GetErrorCodeFromHRESULT(hrCreateCIM);
+                    errorDesc = L"Failed creation of CIM file. HRESULT " + errorCode + L".";

```

Comment:
Same can be done here. Basically, the idea is to not write same kind of text twice in a block. Let's try to rephrase the message wherever needed and try and use same error message that will be shown to the user.

Reply:
As discussed, all the Telemetry variables will have the same Error Message as the actual Console statements. The change for outputting the Error Message on Console from a variable will be done as part of Error Message Improvement Task that will be taken later.